### DISCUSSION TITLE
Amélioration !top et !topweek

### DISCUSSION THREAD

**Discussion by Surdidi** [Crownicles/Crownicles#883] [Nouvelles idées]
ça serait intéressant d'ajouter certaines fonctionnalités aux commandes de top comme par exemple un émoticône pour nous rediriger directement à la page correspondant à notre classement.
Voir des flèches pour nous diriger de pages en pages, mais c'est un peu bloquant d'un point de vue technique.

**Comment by BastLast** [comment]
j'aime

(enfin le coup des flèches pour changer de pages c'est chaud, voir impossible à faire mais la réac pr aller à la bonne page ca peut se faire)

**Reply by Arkitu** [reply]
Les flèches pour changer de page Draftbot Assistant le fait donc c'est pas impossible. Il suffit de mettre le numéro de page dans l'embed et de le lire quand on veut changer de page. Après peut être que ça demanderai plus de ressources, je sais pas exactement comment c'est géré dans Draftbot

**Reply by Arkitu** [reply]
Et même avec des réactions j'ai vu des bots (comme Mudae par exemple) le faire

**Comment by romain22222** [comment]
le cout des fleches, c'est pas possible tant qu'on est pas passés aux slash commands, donc 👋

sinon l'emote pour aller à notre page de classement c'est bien et j'approuve

une autre idée possible serait (grace a un emote) de pouvoir basculer entre les différents tops, si on est hors de notre page, ca redirige vers la 1e, sinon ca nous mets sur notre page de l'autre top

**Comment by Chevalier2** [comment]
Idée oubliée j'ai l'impression. 👀

**Reply by BastLast** [reply]
Je pense qu'on va passer aux boutons à termes
